infra/cifuzz/actions/run_fuzzers/action.yml

```diff
@@ -15,6 +15,10 @@ inputs:
   sanitizer:
     description: 'The sanitizer to run the fuzzers with.'
     default: 'address'
+  run-fuzzers-mode:
```

Comment:
Can you add comments on the two modes and their purpose.

Reply:
Done.